### DISCUSSION TITLE
Cant Find System ApiKey in backoffice !!

### DISCUSSION THREAD

**Discussion by Nass400** [alfio-event/alf.io#1105] [General]
Hi,
im writing to report a problem with :
![image](https://user-images.githubusercontent.com/97454749/187266876-aa8cceaa-a82b-4723-9ba9-14f8969fb478.png)

I cant find the System API Key as shown in the Image above, I can only create or view Organization level APIs.
PS: Im connected as ADMIN

**Comment by cbellone** [comment]
Hi, 
System api key has not been released yet. Will be released soon.
The documentation has been updated so that the people working on other systems and integrating them with alf.io can test their integration against the new APIs before release.

If you need to test it, you have to build a version from the [2.0-M4-maintenance](https://github.com/alfio-event/alf.io/tree/2.0-M4-maintenance) branch
